src/java/org/apache/cassandra/service/ActiveRepairService.java

```diff
@@ -265,6 +264,18 @@ public int getRepairSessionSpaceInMegabytes()
         return stats;
     }
 
+    @Override
+    public void setRepairSessionSpaceInMebibytes(int sizeInMebibytes)
```

Comment:
I wonder if we should take a string with units here instead of fixing the unit? Not a big deal, but since this becomes public API after this releases, might as well ask :)

Reply:
I think the idea was to have the right function with the right expectation from the name that it Is actually mebibyte and not megabytes.  `setRepairSessionSpaceInMegabytes` and `getRepairSessionSpaceInMegabytes` were deprecated. And I didn't get the new format because we opted out of adding new JMX functions for now in favor of potential move to virtual tables when the opportunity to update config from virtual tables land.

Reply:
> we opted out of adding new JMX functions for now in favor of potential move to virtual tables

WFM